tests/test_google_sheets_discovery.py

```diff
@@ -118,11 +118,13 @@ def test_run(self):
                     expected_automatic_fields.remove('modifiedTime')
                 self.assertSetEqual(expected_automatic_fields, actual_automatic_fields)
 
-                # verify columns missing headers or missing values where __sdc_row = 2
+                # verify missing values where __sdc_row = 2
                 # are marked with inclusion of unsupported
-                # BUG_TDL-14475 | https://jira.talendforge.org/browse/TDL-14475
-                failing_streams = {'sadsheet-column-skip-bug', 'Item Master'}  # BUG_TDL-14475
-                if stream not in failing_streams:  # BUG_TDL-14475
+                # The card TDL-14475 was only about adding unsupported 
```

Comment:
I reviewed this ticket. The original writeup for these bugs did not properly cover the two types of issues that we found here. The original assertion is correct in that both streams should be marked as unsupported, but for different reasons. By removing the bug mark and altering the assertion comment the bug is being covering up for the `Item Master` stream. In the future please raise discrepancies between tickets and tests so that issues do not get missed. I will card this out separately.

Reply:
Sure @kspeer825 , we will take care from the next time. Also, we have started looking into the card that you have created.